README.md

```diff
@@ -67,89 +73,93 @@ After getting everything running, you can open the **Pleuston Frontend** applica
+`--no-aquarius`             | Start up Ocean without an instance of `aquarius` and `mongodb`.
+`--no-brizo`                | Start up Ocean without an instance of `brizo`.
+`--no-secret-store`         | Start up Ocean without an instance of `secret-store`.
+`--local-ganache-node`      | Runs a local ganache node.
+`--local-spree-node`        | Runs a local node of the `spree` network.
+`--local-nile-node`         | Runs a node of the `nile` network and connects to the `nile` network.
+`--local-kovan-node`        | Runs a light node of the `kovan` network and connects to the `kovan` network.
+`--reuse-ganache-database`  | Runs the ganache node with a persistent database.
+`--purge`                   | Removes the containers, volumes, artifact folder and networks used by the script.
 
-For example, if you do:
+## Docker Building Blocks
 
-```bash
-./start_ocean.sh --latest
-```
+Ocean compose consist of a set of building Blocks that can be combined to form a local test environment.
 
-then the main/default [docker-compose.yml](docker-compose.yml) will be used, so the following Docker images will all be started:
+### Keeper Node
 
-- mongo:3.6
-- oceanprotocol/keeper-contracts:latest
-- oceanprotocol/aquarius:latest
-- oceanprotocol/brizo:latest
-- oceanprotocol/pleuston:latest
+Controlled by the `--local-*-node` config switches will start a container `keeper-node` that uses port `8545` to expose an rpc endpoint to the Ethereum Protocol. Internal Url: `http://keeper-node:8545`.
 
-To see what ports each of those listens on, read [docker-compose.yml](docker-compose.yml). Note that `keeper-contracts` runs a local Ganache node (not a local Parity Ethereum POA node).
+Hostname      | External Port | Internal Url            | Local Url             | Description
+--------------|---------------|-------------------------|-----------------------|--------------
+`keeper-node` | `8545`        | http://keeper-node:8545 | http://localhost:8545 | An Ethereum RPC node
 
-If you do:
+This node can be one of the following implementations:
 
-```bash
-./start_ocean.sh --no-pleuston
-```
+Node      | Description
+----------|-------------
+`ganache` | Runs a local `ganache-cli` node that is not persistent by default. The contracts from the desired `keeper-contracts` version will be deployed upon launch of this node.
+`spree`   | Runs a local instance of the `spree` network. See [Spree Network](#spree-network) for details.
+`nile`    | Runs a instance of the `nile` network and connects to the Nile testnet. See [Nile Network](#nile-network) for details.
+`kovan`   | Runs a instance of the `kovan` network and connects to the Kovan testnet.
 
-then [docker-compose-no-pleuston.yml](docker-compose-no-pleuston.yml) will be used, so these images will be started:
+### Aquarius
 
-- mongo:3.6
-- oceanprotocol/keeper-contracts:stable
-- oceanprotocol/aquarius:stable
-- oceanprotocol/brizo:stable
+Controlled by the `--no-aquarius` config switch will start two containers:
```

Comment:
that seems confusing, `--no-aquarius` will start Aquarius? Same for the following components where it says the `--no-*` switches will start that component.

Reply:
Rephrased of all the sections to `This block starts what by default, it can be disabled with --no-*`